pyfftw/pyfftw.pxd

```diff
@@ -403,3 +403,65 @@ cdef enum:
     FFTW_PATIENT = 32
     FFTW_ESTIMATE = 64
     FFTW_WISDOM_ONLY = 2097152
+
```

Comment:
Do we need all these attributes to be exposed?

Reply:
Previously, the cdef class FFTW was not declared in a *pxd file. This meant I could not type cimport the associated type from outside the library. With this declaration, now I can.

As for the attributes themselves, as far as I can tell (from Cython complaining), you need all of them to be present in the declaration, as well as all cdef / cpdef methods. def methods don't have to be there.